plugins/woocommerce/src/Blocks/BlockTypes/ProductStockIndicator.php

```diff
@@ -45,7 +46,7 @@ protected function get_block_type_uses_context() {
 	 * @return array
 	 */
 	protected function get_product_types_without_stock_indicator() {
-		return array( ProductType::EXTERNAL, ProductType::GROUPED, ProductType::VARIABLE );
+		return array( ProductType::EXTERNAL, ProductType::GROUPED );
```

Comment:
Variable products only display a stock indicator under the circumstances outlined in this PR. For this reason, we need to remove it from this list.